### ISSUE TITLE
Translation - get all content translated

### ISSUE THREAD

**Issue by ifsimicoded** [cityofaustin/techstack#112]
includes headers, 311, footer, links, static content (not in CMS)

**Comment by amenity** [comment]
Here is the CMS content:                 https://drive.google.com/drive/folders/1rgRRytxDq6KD14kce8I6BdXGJkN87TJA?usp=sharing

**Comment by amenity** [comment]
@ashleecoa can you size this guy and give a description/status update plz

**Comment by ashleecoa** [comment]
@amenity This needs some discussion with the larger team. @courtneyjacinic mentioned that it might not make sense to translate static content like the menu etc as this information might change significant from testing. It would be good to discuss with Content and UX. However, if we proceed. I'd say this is likely a Large

**Comment by amenity** [comment]
@ashleecoa, would it be a Large for you, or for the translators? @atx-adam thought he might have paid for static content translation already. 
Will y'all not want the header, footer, menu, etc. translated for your language usability tests? 🤔

**Comment by ashleecoa** [comment]
@atx-adam paid for the services content @amenity but not static content. We're going to meet with the UX team to discuss that decision on translating static content. Will followup after that.

**Comment by ashleecoa** [comment]
Content: https://airtable.com/shrgmOtwG1wvVEX0V/tblWg4IxNd2bIXvv3
Mockups: https://drive.google.com/drive/folders/1aus-SlYpyOv8_x9ZS29LRHmsQaqx3xd-?usp=sharing

**Comment by ifsimicoded** [comment]
sorrrrry for all the updates -- I was confusing this issue for the dev task. my b.

**Comment by ashleecoa** [comment]
Non-CMS Content (Spanish and Arabic)  @ifsimicoded @mateoclarke 

@cthibaultatx said she can help 

Spanish: https://drive.google.com/file/d/1bzGU_CrivRqb_wIiNJLVamFEGITDtn9-/view?usp=sharing
Arabic: https://drive.google.com/file/d/1548qvXu2JK0Sma5c4rhajdveT6SCV1J2/view?usp=sharing

**Comment by ashleecoa** [comment]
- [ ] Get 3-1-1 content translated
